README.md

```diff
@@ -57,33 +54,29 @@ Use the following URI for GET and DELETE operation on the Restore Point resource
 https://management.azure.com/subscriptions/{SubscriptionID}/resourceGroups/{ResourceGroupName}/providers/Microsoft.Compute/restorePointCollections/{RestorePointCollectionName}/RestorePoints/{RestorePointName}?api-version=2021-03-01
  
 For update of an existing Restore Point Collection resource, only update of tags is permitted.
-## Create Disk
 
+## Create Disk
 1. Get the Disk Restore ID by either using GET call on Restore Point Collection and expanding Restore Points or by doing a call on Restore Point(s). 
-2. You can then use this ARM template to create a disk using diskRestorePoint: https://github.com/Azure/Virtual-Machine-Restore-Points/blob/main/createDiskFromDiskRestorePoint.json
```

Comment:
Why are we making changes to RestorePoints documentation?

Reply:
Restore Points team is not planning to update the GitHub repo anymore as they have the public documentation for APIs published for public preview. So added link to there public here for completeness.